src/spin_sdk/sqlite.py

```diff
@@ -1,8 +1,30 @@
+"""Module for interacting with an SQLite database"""
+
 from typing import List
 from spin_sdk.wit.imports.sqlite import Connection
 
 def open(name: str) -> Connection:
+    """Open a connection to a named database instance.
+
+    If `database` is "default", the default instance is opened.
+
+    A `spin_sdk.wit.types.Err(spin_sdk.wit.imports.sqlite(ErrorAccessDenied)` will be raised when the component does not have access to the specified database.
+
+    A `spin_sdk.wit.types.Err(spin_sdk.wit.imports.sqlite(ErrorNoSuchDatabase)` will be raised when the host does not recognize the database name requested.
+    
+    A `spin_sdk.wit.types.Err(spin_sdk.wit.imports.sqlite(ErrorInvalidConnection)` will be raised when the provided connection is not valid.
+    
+    A `spin_sdk.wit.types.Err(spin_sdk.wit.imports.sqlite(ErrorIo(str))` will be raised when implementation-specific error occured (e.g. I/O)
+    """
     return Connection.open(name)
 
 def open_default() -> Connection:
+    """Open the default store.
+
+    A `spin_sdk.wit.types.Err(spin_sdk.wit.imports.sqlite(ErrorOther(str))` will be raised when the component does not have access to the default database.
+
+    A `spin_sdk.wit.types.Err(spin_sdk.wit.imports.sqlite(ErrorInvalidConnection)` will be raised when the provided connection is not valid.
```

Comment:
I don't _think_ this one is possible, since there's always going to be a default database (which the component may or may not have access to), correct?

Reply:
Oh yes. I will update.